client/src/views/Tests/EditTest.tsx

```diff
@@ -18,6 +18,18 @@ const EditTest: React.FC = () => {
         failureEmail: { body: "", subject: "" },
     });
 
+    const onChange = ({ questions }: Partial<Pick<models.dtos.TestDto, "questions">>) => {
+        ///questions are not undefined or empty
+        if (!questions || !(questions?.length > 0)) {
+            console.log("empty questions");
+        } else {
```

Comment:
This is not needed, since you have the `onValidityChange` that should check the validity of the component independentily